packages/paste-website/src/pages/blog/2023-04-13-paste-newsletter.mdx

```diff
@@ -0,0 +1,185 @@
+export const meta = {
+  title: 'Paste Design System Newsletter - April 2023 Edition',
+  slug: '/blog/2023-04-13-paste-newsletter/',
+  date: '2023-04-13',
+  author: 'Paste',
+  avatar: 'https://avatars.githubusercontent.com/t/3335688?s=280&v=4',
+  status: 'published',
+  excerpt: "What's new in Paste in April 2023? New components, new design language, and more!",
```

Comment:
suggestion: tweak the description/preview a bit so it doesn't look copy pasted 
<img width="851" alt="image" src="https://user-images.githubusercontent.com/75342690/233142323-45eb5a87-af70-43b6-99bf-4017da221133.png">

Reply:
Updated